create_tarball.sh

```diff
@@ -43,13 +43,13 @@ if [ -d ${pilot_version}/software/${os}/${cpu_arch_subdir}/.lmod ]; then
 fi
 if [ -d ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules ]; then
     # module files
-    find ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules -type f >> ${files_list}
+    find ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules -type f | grep -v '/\.wh\.' >> ${files_list}
```

Comment:
Wonder if it would make sense to collect a list of files which have been filtered out and add that to the metadata when uploading the tarball? This would allow the human overseeing the ingestion procedure to double-check if anything had been removed that shouldn't.

Reply:
These `.wh*` files are temporary files created by `fuse-overlayfs`. Although we see them when compiling the list of files to include in the tarball, they're no longer there by the time we create the tarball.
Also, they're useless to us anyway...